.rubocop.yml

```diff
@@ -1,12 +1,12 @@
-AllCops:
-  TargetRubyVersion: '2.2.0'
+require: 
+  - rubocop-rspec
+  - rubocop-performance
 
-Naming/FileName:
+AllCops:
+  # Ruby 2.7 is the MINIMUM Ruby version supported
+  TargetRubyVersion: '2.7.0'
   Exclude:
-    - 'lib/omniauth-mlh.rb'
+    - vendor/**/*
 
-Metrics/BlockLength:
-  Exclude:
-    - 'Rakefile'
-    - '**/*.rake'
-    - 'spec/**/*'
+RSpec/MultipleExpectations:
```

Comment:
I actually dislike this rule. I think we should disable it. This is a good default to include in OHQ as well!

```
RSpec/MultipleExpectations:
  Description: Allow tests to contain multiple expectations
  Enabled: false
```